src/app/listener.ts

```diff
@@ -42,6 +45,38 @@ export class Listener {
 
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
 
+      const commands = Object.entries(this.container.pluginService.plugins)
+        .filter((entry) => {
+          const [, plugin] = entry;
+          return isSlashCommand(plugin);
+        })
+        .map((entry) => {
+          const [name, plugin] = entry;
+          const options = isSlashCommand(plugin)
+            ? (plugin as unknown as ISlashPlugin).parameters
+            : undefined;
+
+          return {
+            name: name,
+            description: plugin.description.substring(0, 99),
```

Comment:
99?

Reply:
Slash commands have a description string length limit of 99 chars.